docs/content/config/environment.md

```diff
@@ -605,9 +605,26 @@ Enable or disable `getmail`.
 
 - **5** => `getmail` The number of minutes for the interval. Min: 1; Max: 30; Default: 5.
 
-#### LDAP
 
+#### OAUTH2
+
+##### ENABLE_OAUTH2
+
+- **empty** => OAUTH2 authentication is disabled
+- 1 => OAUTH2 authentication is enabled
+
+##### OAUTH2_CLIENT_ID
+
+- => Specify the OAuth2 client ID
 
+##### OAUTH2_CLIENT_SECRET
+
+- => Specify the OAuth2 client secret
+
+##### OAUTH2_INTROSPECTION_URL
+
+- => Specify the user info endpoint URL of the oauth2 provider. E.g. `https://oauth2.example.com/userinfo/`, where the trailing slash is MANDATORY (at least for Authentik)
```

Comment:
```suggestion
- => Specify the user info endpoint URL of the oauth2 provider. E.g. `https://oauth2.example.com/userinfo/`, where the trailing slash is MANDATORY (at least for Authentik)

```

Reply:
I'm not sure if we need to document the trailing slash. I believe this will vary by service.

It's a common feature with reverse proxies to support enforcing it or supporting a route without it. That could probably front Authentik too in a way that the reverse proxy supports either and handles any explicit requirement when handing off to Authentik.

It's also not always the `userinfo/` endpoint as I've found.

---

I'll come back to this after this review pass as I'm more inclined to drop these ENV.

Reply:
The trailing slash comment can be left to the guide docs. I'm suggesting with `eg` since I think that's used elsewhere even though it's not entirely correct compared to `e.g.:` 😅 

```suggestion
- => Specify the user info endpoint URL of the oauth2 provider (_eg: `https://oauth2.example.com/userinfo/`_)
```